username_api.py

```diff
@@ -15,6 +15,11 @@
 
 cache = SimpleCache()
 
+session = r.Session()
```

Comment:
please add a comment above stating the need to use it. It will be useful for the developers in future